lib/streams/documentStream.js

```diff
@@ -28,7 +28,22 @@ function createDocumentStream(id_prefix, stats) {
         if (record.POSTCODE) {
           addrDoc.setAddress( 'zip', record.POSTCODE );
         }
-
+        
+        if (useOaAdminFields) {
+          if (record.CITY) {
+            addrDoc.parent.locality.push(record.CITY);
```

Comment:
So after thinking about (and forgetting about, sorry!) this PR for a while, it feels like this is ok as is. Since this is really just an attempt to make things a little better for the case when admin_lookup isn't in use, if things aren't quite perfect it should be ok. If there are bugs, we will be telling people to turn on admin lookup anyway. @trescube what do you think?